package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@auth0/nextjs-auth0": "^3.5.0",
+    "@buf/safedep_api.bufbuild_es": "^1.10.0-20241230154048-f72c3643452e.1",
```

Comment:
@kunaldevxxx You should add PR comment at the line of change and not at the file level.

This package gives us TypeScript type information for our Malware Analysis API from https://buf.build/safedep/api/docs/main:safedep.services.malysis.v1

Using this, we don't need to manually define `type` or `interface` for handling the malware analysis data. We are using auto-generated Typescript safe code to handle response from Malware Analysis service.